src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,61 @@
+package nextstep.courses.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Session {
+
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+    private SessionType sessionType;
+    private Image coverImage;
+    private SessionStatus sessionStatus;
+    private int maximumEnrollment = 0;
+    private int fee;
+    private List<NsUser> participants = new ArrayList<>();
+
+    public Session(LocalDateTime startDate, LocalDateTime endDate, SessionType sessionType, Image coverImage, SessionStatus sessionStatus, int maximumEnrollment, int fee) {
+
+        inputValidation(startDate, endDate, sessionType, coverImage, sessionStatus, maximumEnrollment, fee);
+
+        this.startDate = startDate;
+        this.endDate = endDate;
+        this.sessionType = sessionType;
+        this.coverImage = coverImage;
+        this.sessionStatus = sessionStatus;
+        this.maximumEnrollment = maximumEnrollment;
+        this.fee = fee;
+    }
+
+    private void inputValidation(LocalDateTime startDate, LocalDateTime endDate, SessionType sessionType, Image coverImage, SessionStatus sessionStatus, int maximumEnrollment, int fee) {
+        if(sessionType == SessionType.FREE && maximumEnrollment != 0){
```

Comment:
```suggestion
        if(sessionType.isFree() && maximumEnrollment != 0){
```

객체에게 물어보기 😄 
다른곳도 개선할 부분이 없는지 고민해보아요~